tests/tools/feature_ranking/test_rank_features_groups.py

```diff
@@ -287,6 +293,7 @@ def test_rank_adata_immutability_property(self, field_to_rank):
         adata = read_csv(
             dataset_path=f"{_TEST_PATH}/dataset1.csv", columns_x_only=["station", "sys_bp_entry", "dia_bp_entry"]
         )
+        ep.ad.infer_feature_types(adata, output=None)
```

Comment:
Why do we always need the explicit calls here?

Reply:
Good point - we don't. That still remained from when we didn't automatically call `ep.ad.infer_feature_types`. I removed it from all tests where it isn't necessary to explicitly call it.